crowdsec-docs/sidebars.js

```diff
@@ -372,10 +372,15 @@ module.exports = {
             type: "category",
             label: "Expr",
             items: [
-                "expr/helpers",
-                "expr/event",
+                "expr/intro",
```

Comment:
You don't need intro here anymore when clicking on the main helpers option it will show this instead